Tools/autotest/sim_vehicle.py

```diff
@@ -460,23 +475,23 @@ def options_for_frame(frame, vehicle, opts):
     if ret is None:
         ret = {}
 
-    if not ret.has_key("model"):
+    if "model" not in ret:
         ret["model"] = frame
 
-    if not ret.has_key("sitl-port"):
+    if "sitl-port" not in ret:
         ret["sitl-port"] = True
 
     if opts.model is not None:
         ret["model"] = opts.model
 
     if (ret["model"].find("xplane") != -1 or
-        ret["model"].find("flightaxis") != -1):
-        ret["sitl-port"] = False
+            ret["model"].find("flightaxis") != -1):
+            ret["sitl-port"] = False
```

Comment:
I'm confused here. Why does the instruction inside the if have its indentation increased? The second part of the if I understand (although I would prefer to have the if all in one line), but the instruction inside should have the same indentation as do the previous/next ifs.